cmd/aws/main.go

```diff
@@ -69,6 +69,7 @@ var (
 	rejectExtensions           = flag.String("reject_extension", "", "A list of X.509 extension OIDs, in dotted string form (e.g. '2.3.4.5') which, if present, should cause submissions to be rejected.")
 	signerPublicKeySecretName  = flag.String("signer_public_key_secret_name", "", "Public key secret name for checkpoints and SCTs signer")
 	signerPrivateKeySecretName = flag.String("signer_private_key_secret_name", "", "Private key secret name for checkpoints and SCTs signer")
+	checkpointInterval         = flag.Duration("checkpoint_interval", -1*time.Second, fmt.Sprintf("Interval between checkpoint publishing. If unset, the Tessera default value is %s. Minimum value is 1s.", tessera.DefaultCheckpointInterval))
```

Comment:
Same comments here: what's the rationale for the 1s minimum value, and did you mean to leave %s?

Reply:
The minimum value `1s` comes from Tessera AWS storage implementation. However, that's not an exported const.

https://github.com/transparency-dev/tessera/blob/29298ee9f33e6513a77b117f73d9d6ed6538865a/storage/aws/aws.go#L73